### PR TITLE
feat: Dockerfile - Add YData image

### LINKED ISSUE #25: Dockerfile - Add YData image

### PR DESCRIPTION
**Pull request by victoriavilasb**
Closes #25

### CODE REVIEW

File: data-science/ydata/cpu/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# YData
+https://storage.googleapis.com/pip.bucket.ydata-fronteira-754.ydata.ai/prerelease/ydata-0.6.1.tar.gz?x-goog-signature=463bde798e95c02e9d206c25a16a8e6ed788a608f1aed248824c048300a598223a237c29e644a0b4bcee297933296344b8fb965447c5e6d25e21107086cc9a3b96befd9633ec325c169b6d89e0a9760d3e01c59bbedc5ff26c0e776847992ae4b94502614a904fa3c8dce57b11da81c0ced3837c45bbfb3cd2b4e2046c6ac214b4a4737d6a2d7d1b04b539ab6ca63a62826e28bd77eecaca1d4d33dcb02f7d0510e9cf5bdd7d3a34f914d7e79b8a670199a892331b7f1540abdcfff1e2dc749029b4d285214a8363aecef265fb74778c36c80095f6b20c38ef291cfe647dfa4f8f4b76bb85e57f251800101db3a9d9740b0d1e4c39f27c86668668943d58ad5f&x-goog-algorithm=GOOG4-RSA-SHA256&x-goog-credential=pip-bucket-sa%40ydata-fronteira-754.iam.gserviceaccount.com%2F20201202%2Feu%2Fstorage%2Fgoog4_request&x-goog-date=20201202T100840Z&x-goog-expires=604800&x-goog-signedheaders=host
```
Comment (portellaa) [inline data-science/ydata/cpu/requirements.txt:2]: we should migrate to git based url

File: data-science/ydata/cpu/build.args [hunk 2]
```diff
@@ -0,0 +1,2 @@
+TAG=0.1.2-cpu
```
Comment (portellaa) [inline data-science/ydata/cpu/build.args:1]: this image has to be based on gpu

File: data-science/ydata/cpu/build.args [hunk 3]
```diff
@@ -0,0 +1,2 @@
...
+JUPYTERLAB_VERSION=1.2.15
```
[the remark this answers is not in the document]
Comment (portellaa) [reply data-science/ydata/cpu/build.args:2]: this will install the required packages for it

[the remark this answers is not in the document]
Comment (rafaelcpalmeida) [reply data-science/ydata/cpu/build.args:2]: Not TensorFlow 2 🤔

File: data-science/ydata/cpu/build.args (hunk 2 above)
Comment (victoriavilasb) [reply data-science/ydata/cpu/build.args:1]: ok

**Review by portellaa — COMMENTED** [review]
just wait for jupyterlab cuda powered image

**Review by portellaa — APPROVED** [review]

**Pull request merged**